src/store/theme/saga.ts

```diff
@@ -0,0 +1,17 @@
+import { takeLatest, put } from 'redux-saga/effects';
+import { SagaActionTypes, receive } from '.';
+import { ViewModes } from '../../shared-components/theme-engine';
+
+export const savedViewMode = 'viewMode:isLight';
+
+export function* setViewMode(action) {
+  const viewMode = action.payload;
+
+  localStorage.setItem(savedViewMode, viewMode === ViewModes.Dark ? 'false' : 'true');
```

Comment:
```suggestion
  localStorage.setItem(savedViewMode, viewMode === ViewModes.Dark);
```

Reply:
Argument of type 'boolean' is not assignable to parameter of type 'string'